karman-avionics/src/utils/tc_fixes.h

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
You are a god king for just casually doing this much stuff so quickly....
Thanks